### ISSUE TITLE
Fails in browser

### ISSUE THREAD

**Issue by dudewad** [frysztak/markdown-to-quill-delta#1]
Doesn't work in-browser (bundled this with an Angular build) because of
```
Uncaught (in promise): ReferenceError: process is not defined
ReferenceError: process is not defined
```
`process` is used by `path.js` to check for OS (windows). 
Is this intended to be used only server-side? Quill is browser-based so not being able to do this on the client seems strange. Am I using it incorrectly? Doesn't seem like there's much to do  other than importing it (This is a Typescript project)

My usage is quite simple:

```Typescript
import markdownToDelta from 'markdown-to-quill-delta';

// ...
this.quill.setContents(markdownToDelta(markdown));
```

**Comment by frysztak** [comment]
Hi,

perhaps I'm missing something, but I think that `path` is only used in tests (which are not meant to run in the browser). `process` is not imported anywhere in the project, so I'm really not sure how you're getting this error message. Can you share more info?

**Comment by leo-pires** [comment]
Hello! Same error here.

The error comes from `unified().use(markdown)` in `https://github.com/frysztak/markdown-to-quill-delta/blob/master/src/markdownToDelta.ts#L7`.

**Comment by Filipeue** [comment]
@frysztak   
here is output from webpack build

```
ERROR in ./node_modules/markdown-to-quill-delta/node_modules/vfile/core.js 3:11-26
Module not found: Error: Can't resolve 'path' in '...\node_modules\markdown-to-quill-delta\node_modules\vfile'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
        - install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:  
        resolve.fallback: { "path": false }
resolve 'path' in '...\node_modules\markdown-to-quill-delta\node_modules\vfile'
  Parsed request is a module
  using description file: ...\node_modules\markdown-to-quill-delta\node_modules\vfile\package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      ...\node_modules\markdown-to-quill-delta\node_modules\vfile\node_modules doesn't exist or is not a directory
      ...\node_modules\markdown-to-quill-delta\node_modules\node_modules doesn't exist or is not a directory
      looking for modules in ...\node_modules\markdown-to-quill-delta\node_modules
        single file module
          using description file: ...\node_modules\markdown-to-quill-delta\package.json (relative path: ./node_modules/path)
            no extension
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\markdown-to-quill-delta\node_modules\path doesn't exist
            .tsx
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\markdown-to-quill-delta\node_modules\path.tsx doesn't exist
            .ts
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\markdown-to-quill-delta\node_modules\path.ts doesn't exist
            .jsx
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\markdown-to-quill-delta\node_modules\path.jsx doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\markdown-to-quill-delta\node_modules\path.js doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\markdown-to-quill-delta\node_modules\path.json doesn't exist
            .wasm
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\markdown-to-quill-delta\node_modules\path.wasm doesn't exist
        ...\node_modules\markdown-to-quill-delta\node_modules\path doesn't exist
      ...\node_modules\node_modules doesn't exist or is not a directory
      looking for modules in ...\node_modules
        single file module
          using description file: ...\package.json (relative path: ./node_modules/path)
            no extension
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\path doesn't exist      
            .tsx
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\path.tsx doesn't exist  
            .ts
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\path.ts doesn't exist   
            .jsx
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\path.jsx doesn't exist  
            .js
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\path.js doesn't exist   
            .json
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\path.json doesn't exist 
            .wasm
              Field 'browser' doesn't contain a valid alias configuration        
              ...\node_modules\path.wasm doesn't exist 
        ...\node_modules\path doesn't exist
 @ ./node_modules/markdown-to-quill-delta/node_modules/vfile/index.js 4:12-32    
 @ ./node_modules/markdown-to-quill-delta/node_modules/unified/index.js 5:12-28  
 @ ./node_modules/markdown-to-quill-delta/lib/markdownToDelta.js 2:0-35 5:22-29 
```

**Comment by frysztak** [comment]
@Filipeue thanks. I released version [0.7.0](https://github.com/frysztak/markdown-to-quill-delta/releases/tag/v0.7.0) that fixes that
